cgi-bin/LJ/Session.pm

```diff
@@ -950,8 +950,8 @@ sub get_cookie_path {
 
 sub get_path_user {
     my ( $path ) = @_;
-    return unless $path =~ m!^/(\w{1,25})\b!;
-    # FIXME: username length should be a named constant (#1568)
+    my $exp = '^/(\w{1,' . $LJ::USERNAME_MAXLENGTH . '})\b';
+    return unless $path =~ /$exp/;
```

Comment:
I'll take your word for it, but I'm afraid that line is totally illegible to me! A hazard of not using Perl very often ;)